test/test.highlevelConsumer.js

```diff
@@ -0,0 +1,222 @@
+'use strict';
+var util = require('util');
+var EventEmitter = require('events');
+var sinon = require('sinon');
+
+var libPath = process.env['KAFKA_COV'] ? '../lib-cov/' : '../lib/';
+
+// var Client = require(libPath + 'client');
```

Comment:
Yes please. I've been trying to clean up things in the code that don't make sense or isn't used anymore.